test/gst-msdk/encode/10bit/vp9.py

```diff
@@ -0,0 +1,105 @@
+###
+### Copyright (C) 2019-2020 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from ...util import *
+from ..encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+class VP9_10EncoderTest(EncoderTest):
```

Comment:
It looks like we already have class VP9_10EncoderTest  in gst-vaapi folder vp9.py ?

Reply:
@zhoujd this is not a problem.  It is in a different namespace (module).